api/DHSC.FingertipsNext.Modules.Core.Service/LoggingExtensions.cs

```diff
@@ -0,0 +1,9 @@
+using Microsoft.Extensions.Logging;
+
+namespace DHSC.FingertipsNext.Modules.Core.Service;
+
+public static partial class LoggingExtensions
+{
+    [LoggerMessage(EventId = 0, Level = LogLevel.Error, Message = "Search Specific Log Example failed operation during: `{Context}`")]
+    public static partial void LogExampleTwo(this ILogger logger, string context, Exception exception);
```

Comment:
Just for my understanding, it's possible to define log messages in this way that don't take an exception parameter, right? i.e. so we can log non-error messages?

Reply:
You can yes - you would just excluded the exception in the method parameters, e.g.

 public static partial void LogExampleTwo(this ILogger logger, string context)